sys/docker/alpine-phpfpm/Dockerfile

```diff
@@ -41,7 +41,10 @@ RUN apk add --no-cache --virtual .builddeps \
         mbstring \
         opcache \
         zip \
-    && pecl install msgpack yaml redis \
+    && wget https://github.com/FriendsOfPHP/pickle/releases/download/v0.6.0/pickle.phar && mv pickle.phar /usr/local/bin/pickle \
+    && chmod +x /usr/local/bin/pickle \
+    && pickle install yaml@2.1.0 redis --defaults --no-interaction \
+#    && pickle install msgpack \
     && cp /usr/local/etc/php/php.ini-production /usr/local/etc/php/php.ini
```

Comment:
can't be an `ln -nfs /usr/local/etc/php/php.ini-production /usr/local/etc/php/php.ini` ?  It would avoid confusion on which file is the source of truth